eZ/Publish/Core/MVC/Legacy/Tests/Image/AliasCleanerTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * File containing the AliasCleanerTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\MVC\Legacy\Tests\Image;
+
+use eZ\Publish\Core\MVC\Legacy\Image\AliasCleaner;
+use PHPUnit_Framework_TestCase;
+
+class AliasCleanerTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var AliasCleaner
```

Comment:
Nitpick: we usually favour fully qualified namespaces in phpdoc, for when you don't have a smart IDE.